TFNativeOp.py

```diff
@@ -402,14 +404,23 @@ def _make_mod(self):
     # right symbols (e.g. the `sgemm_` symbol).
     ld_flags = []
     have_blas_lib = False
-    if self.search_for_runtime_blas:
+
+    if self.blas_lib is not None and os.path.exists(self.blas_lib):
+      path = os.path.dirname(self.blas_lib)
+      if path == "":
+        path = "."
+      ld_flags += ["-L%s" % path, "-l:%s" % os.path.basename(self.blas_lib)]
+      have_blas_lib = True
+    if not have_blas_lib and self.search_for_runtime_blas:
       import Util
       libs = Util.find_sgemm_libs_from_runtime()
       if libs:
         numpy_libs = [fn for fn in libs if "/numpy/.libs/" in fn]
         if numpy_libs:
           # Prefer Numpy; move to front.
           libs = numpy_libs + [fn for fn in libs if fn not in numpy_libs]
+        if self.blas_lib is not None:
+          libs = filter(lambda l: self.blas_lib in l, libs)
```

Comment:
Instead of `filter`, I would use list comprehensions, which makes it easier to read.
But also, I would then add an `assert` that there is actually such a lib.
I.e.:
```suggestion
          libs = [l for l in libs if self.blas_lib in l]
```

Reply:
Ah, wait, I see, you might later add more to `libs`. So ignore the `assert`.

Reply:
See also below. Do the filtering just in the end, and then also add the `assert libs` there.